docs/index.json

```diff
@@ -202,6 +202,14 @@
 			"on": "Collections",
 			"purpose": "How many items should the collection start with? If not provided, it is 1, unless `mv-like` is also specified, in which case it's 0 (to prevent infinite recursion)."
 		},
+		{
+			"name": "mv-uploads",
+			"on": "Mavo root",
+			"purpose": "Provide different upload backend than storage backend.\n**Note:** For now, it is only available in [the development version](https://mavo.io/get/).",
+			"live": "yes",
```

Comment:
Is it actually live? Like, if you set it with an expression does it update?

Reply:
Yep! I made a [simple pen](https://codepen.io/dmitrysharabin/pen/LYpEBeZ) to prove it. 😊Please, have a look.

Reply:
Nice!